src/jit/lclvars.cpp

```diff
@@ -1769,6 +1789,7 @@ void Compiler::lvaPromoteLongVars()
         {
             if (lvaGetPromotionType(varDsc->lvParentLcl) != PROMOTION_TYPE_INDEPENDENT)
             {
+                JITDUMP("Not promoting long local V%02u: field of dependently-promoted struct\n", lclNum);
```

Comment:
It seems like this should be an NYI - is there any scenario where this *wouldn't* be an issue? (I suppose there could be only references that don't matter, like some arg/return scenarios, but it doesn't seem worth identifying those and letting them go through.)

---
In reply to: [77034863](https://github.com/dotnet/coreclr/pull/7000#discussion_r77034863) [](ancestors = 77034863)